zuul.yaml

```diff
@@ -6,6 +6,21 @@
     pre-run: zuul.d/pre.yaml
     run: zuul.d/run.yaml
     post-run: zuul.d/fetch-logs.yaml
+    vars:
+      wazo_debian_repo: main
+      wazo_distribution: wazo-dev-buster
+
+- job:
+    parent: base
+    name: wazo-ansible-uc-archive
+    description: Install Wazo Platform UC archive
+    timeout: 10800
+    pre-run: zuul.d/pre.yaml
+    run: zuul.d/run.yaml
+    post-run: zuul.d/fetch-logs.yaml
+    vars:
+      wazo_debian_repo: archive
+      wazo_distribution: wazo-19.17
```

Comment:
how can test that we have the correct version installed?

Reply:
The proper way should be to check the wazo-platform version
```sh
echo "$(LANG='C' apt-cache policy wazo-platform | grep Installed | grep -oE '[0-9]{2}\.[0-9]+(\.[0-9]+)?' | head -n1)"
```

the easy way:
```sh
cat /usr/share/wazo/WAZO-VERSION
```

Reply:
ok taking care of that using /usr/share/wazo/WAZO-VERSION